### ISSUE TITLE
Add distractors to parsons problems

### ISSUE THREAD

**Issue by igordsm** [insper-education/active-handout-plugins-py#22]
**Proposal**: 

!!! exercise parsons
   description
   ˋˋˋpython
    correct code
   ˋˋˋ

   ˋˋˋpython
   these lines 
   are
   distractors
   ˋˋˋ

@toshikurauchi thoughts on this?

**Comment by toshikurauchi** [comment]
I think it would be nice to support multiple solutions. How about:

````
!!! exercise parsons
    Description

    ```python
    Solution 1
    ```

   ```python
    Solution 2
    ```

    !!! distractor
        ```python
        Distractors
        ```
````

**Comment by igordsm** [comment]
It seems reasonable to mark distractors explicitly inside the exercise. I wouldn't implement alternative solutions yet on the same PR as this one. For me these are two different things and I would rather create more parsons problems before committing to multiple solutions.

**Comment by toshikurauchi** [comment]
Agreed. If we have the explicit marker for distractors the syntax won't break when/if we decide to support alternative solutions.

**Comment by rafaelcorsi** [comment]
I got distracted reading the issue
